tests/pluginbase.test.js

```diff
@@ -1,35 +1,53 @@
 'use strict';
 
 require('should');
+var benv = require('benv');
 
 describe('pluginbase', function ( ) {
 
-  var jsdom = require('jsdom').jsdom;
-  var doc = jsdom('<body></body>');
-  var window = doc.parentWindow;
+  before(function (done) {
+    benv.setup(function() {
+      benv.expose({
+        $: require('jquery')
+        , jQuery: require('jquery')
+        , d3: require('d3')
+        , io: {
+          connect: function mockConnect() {
+            return {
+              on: function mockOn(event, callback) {}
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['callback' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/799298491/issues/source?bid=2452379&fileBranchId=2725007#l17)